experimental/torch_xla2/torch_xla2/ops/jtorch.py

```diff
@@ -98,13 +99,11 @@ def get_params(*a):
   return jnp.einsum(equation, *filtered_operands)
 
 
-def _sdpa_reference(
-   query, key, value, attn_mask=None,
-   dropout_p=0.0, is_causal=False, scale=None) -> torch.Tensor:
+def _sdpa_reference(query, key, value, attn_mask=None, dropout_p=0.0,
+        is_causal=False, scale=None, enable_gqa=False) -> torch.Tensor:
```

Comment:
thanks, do we have short description for `enable_gpa`?

Reply:
yes: https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html
this is newly added in torch 2.5